cpu_features.c

```
@@ -23,3 +23,19 @@ Z_INTERNAL void cpu_check_features(struct cpu_features *features) {
 }
+
+#ifdef HAVE_GETAUXVAL
+#include <sys/auxv.h>
+
+Z_INTERNAL unsigned long zng_getauxval(unsigned long type) {
+    return getauxval(type);
+}
+#elif defined(HAVE_ELF_AUX_INFO)
+#include <sys/auxv.h>
+
+Z_INTERNAL unsigned long zng_getauxval(unsigned long type) {
```

Cppcheck (2.20.0)

[style] The function 'zng_getauxval' is never used.

(unusedFunction)